style.css

```diff
@@ -1,74 +0,0 @@
-body {
-    margin: 20, 0;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    background-color: black;
-}
-
-#title-img {
-    width: 500px;
-    height: 300px;
-}
-
-h3 {
-    font-size: 26px;
-}
-
-p {
-    margin-bottom: 20px;
-}
-
-.game-intro {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    color: #FCC40B;
-}
-
-.btn {
-    display: flex;
-    flex-direction: row;
-}
-
-button {
```

Comment:
I would scope my styles to classes instead of elements where I can... gives you a little more control.